flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,256 @@
+                                                  series_id=tempSeries_id,
+                                                  tvdb_id=request.get('tvDbId'),
+                                                  imdb_id=request.get('imdbId'),
+                                                  ombi_request_id=request.get('id'),
+                                                  ombi_childrequest_id=childRequest["id"],
+                                                  ombi_season_id=season["id"],
+                                                  ombi_episode_id=episode["id"],
+                                                  ombi_approved=episode.get('approved'),
+                                                  ombi_available=episode.get('available'),
+                                                  ombi_requested=episode.get('requested'))
+                                    if entry.isvalid():
+                                        log.debug('Valid entry %s', entry)
+                                        if config.get('only_approved') and not entry.get('ombi_approved'):
+                                            log.verbose('Request not approved skipping: %s', entry.get('title'))
+                                        else:
+                                            if not config.get('include_available') and entry.get('ombi_available'):
+                                                log.verbose('Request already available skipping: %s', entry.get('title'))
+                                            else:
+                                                entries.append(entry)
+                                    else:
+                                        log.error('Invalid entry created? %s', entry)
+                                        continue
+            return entries
+ 
+class OmbiSet(MutableSet):
+    supported_ids = ['imdb_id', 'tmdb_id', 'ombi_id']
+    schema = {
+        'type': 'object',
+        'properties': {
+            'base_url': {'type': 'string'},
+            'port': {'type': 'number', 'default': 3579},
+            'api_key': {'type': 'string'},
+            'type': {'type': 'string', 'enum': ['shows', 'seasons', 'episodes', 'movies']},            
+            'only_approved': {'type': 'boolean', 'default': True},
+            'include_available': {'type': 'boolean', 'default': False},
+            'strip_year': {'type': 'boolean', 'default': False}
+        },
+        'required': ['api_key', 'base_url', 'type'],
+        'additionalProperties': False
+    }
+
+    @property
+    def requests(self):
+        if not self._requests:
+            self._requests = OmbiBase.list_requests(self.config)
+        return self._requests
+        
+    def show_match(self, entry1, entry2):
+        if any(entry1.get(ident) is not None and entry1[ident] == entry2.get(ident) for ident in
+               ['series_name', 'tvdb_id', 'imdb_id']):
```

Comment:
Isn't the point to use supported_ids here?

Reply:
To be honest i don't completely understand this logic as it was pulled over from the trakt plugin.